### DISCUSSION TITLE
Soil UCS test results

### DISCUSSION THREAD

**Discussion by m-sams-artc** [australiangeomechanics/agsformat#43] [Q&A] [answered]
Does the current ags format (with or without AU localisation) support unconfined compressive strength (UCS) test results on soils - typically after stabilisation? It is relatively common in projects to stabilise soils (with lime, cement, flyash, polymer products) and usually UCS tests are done to demonstrate strength gain. Would like a way to have this in gINT, and be able to transfer data in an ags file. Should be columns for stabiliser, dosage, curing time, curing environment, UCS or others as deemed suitable.

**Comment by SolidRockProgrammer** [comment, marked as answer]
There is a Soil UCS group in AGS 4.1.1 - LUCT
one could add custom headings like these from CMPG (and many other groups)
CMPG_STAB	%	2SF	Amount of stabiliser added
CMPG_STYP		X	Type of stabiliser added
